lizmap/modules/lizmap/lib/Project/Repository.php

```diff
@@ -249,4 +267,52 @@ public function getProjects()
 
         return $projects;
     }
+
+    /**
+     * Get the repository projects metadata.
+     *
+     * @return array of project metadata
```

Comment:
`@return ProjectMetada[]`  (or lizmapProjectMetadata for 3.4/3.3

the goal is to help editors ;-)